src/main/java/bio/terra/cli/command/shared/options/ControlledCloningInstructionsForUpdate.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.cli.command.shared.options;
+
+import bio.terra.workspace.model.CloningInstructionsEnum;
+import picocli.CommandLine;
+
+public class ControlledCloningInstructionsForUpdate {
+  /**
+   * Use this --cloning flog for controlled resources.
```

Comment:
sp "flog"

Reply:
Use this --new-cloning flag when updating controlled resources.